src/management-ui/page/courses/list.html

```diff
@@ -17,7 +17,7 @@ <h1 class="heading-large">Courses</h1>
                             <a href="/courses/{course.id}">{course.title}</a>
                         </th>
                         <td>
-                            <a href="{$config.LPG_UI_SERVER}/courses/{course.id}/">Review</a>
+                            <a href="{$config.LPG_MANAGEMENT_URL}/courses/{course.id}/">Review</a>
```

Comment:
This was originally put in as a 'preview' type thing - so view the course on LPG UI. No need to worry about though as new admin coming.